src/main/java/duke/task/TaskList.java

```diff
@@ -0,0 +1,48 @@
+package duke.task;
+
+import java.util.ArrayList;
+import java.util.List;
+
+/**
+ * Represents a list of tasks.
+ */
+public class TaskList {
+    private List<Task> list;
```

Comment:
Perhaps could name this variable as tasks (plural form) to represent the collection of Task objects?